is_iam_enabled.go

```diff
@@ -0,0 +1,74 @@
+package alks
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+)
+
+type IsIamEnabledRequest struct {
+	Account string `json:"account,omitempty"`
+	Role    string `json:"role,omitempty"`
```

Comment:
I also missed this on my first review, but there's already an `AccountDetails` struct you can use here that has an identical format.
```suggestion
	AccountDetails
```